pyproject.toml

```
@@ -22,6 +22,35 @@ extras = dev
 """
 
 [tool.ruff]
+exclude = [
+    ".bzr",
+    ".direnv",
+    ".eggs",
+    ".git",
+    ".hg",
+    ".mypy_cache",
+    ".nox",
+    ".pants.d",
+    ".ruff_cache",
+    ".svn",
+    ".tox",
+    ".venv",
+    "__pypackages__",
+    "_build",
+    "buck-out",
+    "build",
+    "dist",
+    "node_modules",
+    "venv",
+    ".venv",
+    "scripts",
+]
+
```

Most of these are already in the defaults, I recommend to use `extend-exclude` instead (https://docs.astral.sh/ruff/settings/#extend-exclude):

```suggestion
extend-exclude = [
    ".venv",
    "scripts",
]

```